### ISSUE TITLE
Not able to install Calibre CLI in Ubuntu machine

### ISSUE THREAD

**Issue by Rahulvkumar** [calibreapp/cli#94]
## CLI Installation method

* [x] NPM / NPX
* [ ] Binary

## Output of `calibre --version`

root@ip-*****:/home/delops# calibre --version
Traceback (most recent call last):
  File "/usr/bin/calibre", line 19, in <module>
    from calibre.gui2.main import main
  File "/usr/lib/calibre/calibre/gui2/__init__.py", line 21, in <module>
    from calibre.utils.date import UNDEFINED_DATE
  File "/usr/lib/calibre/calibre/utils/date.py", line 13, in <module>
    from dateutil.tz import tzlocal, tzutc, EPOCHORDINAL
  File "/usr/local/lib/python2.7/dist-packages/dateutil/tz/__init__.py", line 2, in <module>
    from .tz import *
  File "/usr/local/lib/python2.7/dist-packages/dateutil/tz/tz.py", line 21, in <module>
    from six.moves import _thread

## Operating system

* [ ] Mac
* [ ] Windows
* [x] Linux

## What you expected to happen
Clean installation of calibre CLI without any issue

## What happened
 I performed "**npm install calibre -g** ", but got the below error. I am not able to install calibre CLI in my Ubuntu machine.

1842 verbose linkStuff   '/usr/local/lib/node_modules/calibre/node_modules' ]
1843 info linkStuff humanize@0.0.9
1844 silly gunzTarPerm extractEntry index.js
1845 silly gunzTarPerm extractEntry readme.md
1846 silly addNameRange number 2 { name: 'samverschueren/stream-to-observable',
1846 silly addNameRange   range: '>=0.3.0-0 <0.4.0-0',
1846 silly addNameRange   hasData: true }
1847 silly addNameRange versions [ undefined, [] ]
1848 error TypeError: Cannot read property 'latest' of undefined
1848 error     at next (/usr/share/npm/lib/cache.js:687:35)
1848 error     at /usr/share/npm/lib/cache.js:675:5
1848 error     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:142:7)
1848 error     at /usr/lib/nodejs/graceful-fs/polyfills.js:133:7
1848 error     at Object.oncomplete (fs.js:107:15)
1849 error If you need help, you may report this log at:
1849 error     <http://github.com/isaacs/npm/issues>
1849 error or email it to:
1849 error     <npm-@googlegroups.com>
1850 error System Linux 3.13.0-43-generic
1851 error command "/usr/bin/nodejs" "/usr/bin/npm" "install" "calibre" "-g"
1852 error cwd /root
1853 error node -v v0.10.25
1854 error npm -v 1.3.10
1855 error type non_object_property_load
1856 verbose exit [ 1, true ]

## Steps to Reproduce (for bugs)
npm install calibre -g 

Thanks in advance,
Rahul

**Comment by benschwarz** [comment]
Hi @Rahulvkumar, 

I can see a combination of a few things that are causing these issues. 

Firstly, it looks like you've installed calibre via Ubuntu apt (or it was already on the system). The ubuntu apt system packages a e-book as "calibre", but this is not the same as the npm JavaScript package.

It also appears as though the node version (seen in the error message) is `0.10.25` ([Released in 2014](https://nodejs.org/uk/blog/release/v0.10.25/), and is no longer maintained). You will need a newer version of node.

The current LTS release of node is `10.16.0`, and there's also a current release `12.6.0`. Either of these will do fine. 

In summary you'll need to: 

* Remove the apt calibre package (`apt-get remove --purge calibre`) 
* Update node to a more recent version
* Install the calibre npm package

Let us know via support if we can help in any other way.
